src/components/Common/Module/FieldPicker.vue

```diff
@@ -1,49 +1,126 @@
 <template>
-    <fieldset class="form-group">
+    <fieldset
+      class="form-group"
+      :disabled="disabled"
+    >
+      <b-form-group
+        :label="$t('block:recordList.fields')"
+        label-cols="2"
```

Comment:
label-cols means it will use horizontal layout, as I can see in the designs that's not how its supposed to look

Reply:
Discussed with Denis. We are going to display it this way so more of the selected/available fields can be seen.